### ISSUE TITLE
Solid the comic

### ISSUE THREAD

**Issue by phochste** [solid/deit#21]
I'm planning to do a "Solvember" comic on Twitter : each day of November a new comic about a Solid topic..based on
my own notes learning about Solid. It is more form a technical viewpoint .. Are there people here who would like to help me a bit to see if I don't make mistakes, drawing/saying things that are not true?
 
Here is an example:

![Untitled_Artwork 7](https://user-images.githubusercontent.com/66416/137845469-3bef0dc7-fc4a-48ed-8022-01e3366bd13a.png)

I'm a part time comic artist ..next to my phd research on decentralised web.

**Comment by csarven** [comment]
Great idea / contribution! Please share far and wide. Happy to review.

**Comment by csarven** [comment]
I'd suggest to not emphasise on "Linked Data Protocol" or "LDP". Perhaps just generally "Linked Data". It is true that Solid Protocol borrows a bit from LDP (and Solid server can be built on top of LDP) but for the most part Solid draws from plain ol' HTTP.

"No need for a server" may be a bit misleading. All applications interact with a server. Perhaps you meant to emphasise that the UI of the application is decoupled from the data on the server.

**Comment by phochste** [comment]
@csarven wow, thanks for the help if you find time. Here is the stash of drawings I already did. I'll update some based on your input this weekend.

https://pod.inrupt.com/hochstenbach/public/solvember/

**Comment by csarven** [comment]
Will go over.

Have you considered what will be the persistent URL for these?

Does this have its own repo? We could create one under solid/ if necessary.

It'd be great to have an ongoing project providing both visual and auditory explanations of Solid. There was some interest / work in the community in this space - names/links completely escape me right now. I'll have to dig around.

@KyraAssaad , what do you think/advise? 

You're welcome to reuse/purpose anything of interest from these visuals:
* https://csarven.ca/linked-research-decentralised-web#figure-centralised-decentralised-architectures
* https://csarven.ca/linked-research-decentralised-web#degree-of-control-identifier-data-application
* https://csarven.ca/linked-research-decentralised-web#examplar-information-space

**Comment by phochste** [comment]
@csarven I'm willing to store them anywhere. I'm part of the solid organisation (but just for a few days). I don't know of the etiquette here if I can just create my own repo, or add things to this repo.

**Comment by KyraAssaad** [comment]
Wow, great work @phochste! I love your drawing style. This would be really great to highlight in the community and have it live somewhere accessible to newcomers. 

With the ongoing redesign of solidproject.org, I believe we can find a place on the site for community-created resources for learning about Solid. Pinging @MarrelleBailey so we can take this into account!

**Comment by jeff-zucker** [comment]
I agree with Kyra and would like to suggest something in addition.  I am in the process of writing a glossary of terms and abbreviations aimed at beginners.  I have ideas for illustrations but no skillz.  Perhaps we could work together to use your drawings as illustrations to accompany the glossary.
